ts/features/itwallet/credentials/store/selectors/index.ts

```diff
@@ -130,3 +130,8 @@ export const itwCredentialsEidStatusSelector = createSelector(
       O.toUndefined
     )
 );
+
+export const itwHasMdlCredentialSelector = createSelector(
+  itwCredentialsByTypeSelector,
+  credentials => O.isSome(credentials[CredentialType.DRIVING_LICENSE] || O.none)
+);
```

Comment:
As I stated in a previous comment, checking for the existence of a credential directly in the `ItwPresentationCredentialDetailScreen` component could simplify the logic thus removing the need for this type of type-specific selectors (which are hardly scalable).